src/Components/Publishing/Fixtures/Helpers.tsx

```diff
@@ -8,3 +10,40 @@ export const EditableChild = type => {
 export const TextFromArticle = article => {
   return compact(map(article.sections, "body")).join("")
 }
+
+export const WrapperWithContext = (context, contextTypes, children) => {
+  const wrapperWithContext = createClass({
```

Comment:
Why use `create-react-class`?

Reply:
`createClass` was removed from the API in React 16, and it threw a message saying to use this package.  Is there another syntax to achieve this without createClass?